src/main/java/com/mg105/entities/Battle.java

```diff
@@ -65,7 +105,23 @@ public int removeChar(BattleCharacter fainted, boolean isOpponent) {
         } else if (this.playerCharacters.isEmpty()) {
             this.battleStatus = -1;
         }
+    }
```

Comment:
You never set battle status 1 is the user has won the battle. I'm not sure if needs to be here or somewhere else but it should be implemented somewhere